### PR TITLE
Update cli-download.rst

### PR DESCRIPTION
**Pull request by nvibert**
Proposed changed to docs to support M1/M2 (Apple Silicon) installation

Signed-off-by: Nico Vibert <nicolas.vibert@isovalent.com>

### CODE REVIEW

**Review by pchaigno — APPROVED** [review]

File: Documentation/gettingstarted/cli-download.rst [hunk 1]
```diff
@@ -12,14 +12,23 @@ various features (e.g. clustermesh, Hubble).
...
+  .. group-tab:: macOS (Intel)
 
     .. code-block:: shell-session
 
       curl -L --remote-name-all https://github.com/cilium/cilium-cli/releases/latest/download/cilium-darwin-amd64.tar.gz{,.sha256sum}
       shasum -a 256 -c cilium-darwin-amd64.tar.gz.sha256sum
       sudo tar xzvfC cilium-darwin-amd64.tar.gz /usr/local/bin
       rm cilium-darwin-amd64.tar.gz{,.sha256sum}
+      
+   .. group-tab:: macOS (Apple Silicon (ARM))
```
Comment (qmonnet) [inline Documentation/gettingstarted/cli-download.rst]:
```suggestion
   .. group-tab:: macOS (Apple Silicon - ARM)
```

Nit: Easier to read than embedded parenthesis?

Alternatively, Documentation/gettingstarted/istio.rst uses `OSX` and `OSX (Apple Silicon)`.

**Review by qmonnet — CHANGES_REQUESTED** [review]
Thanks!

Contents look good, but it seems that the indentation for the new `.. group-tab` is wrong: the new commands are included in the same tab, the ARM tab is empty.

image

Note that you can preview the changes by clicking on the “Netlify preview” link from CI (although I think this is going to change very soon), or you can build locally with `make render-docs`.

**Review by qmonnet — COMMENTED** [review]
Looks great now, thank you! Can you please squash your commits into a single one? (No need to keep the `Co-authored-by` tag for my suggestion here, esp. because GH didn't default to my work email.)

You can ping me or click “re-request review” for me after that to make sure I see the update :)

**Comment by nvibert** [comment]
Screenshot 2022-06-14 at 10 02 49
That looks better.

**Review by qmonnet — APPROVED** [review]
Renders all good this time, thanks!

**Pull request merged**